config/controller.yaml

```diff
@@ -88,10 +88,6 @@ spec:
           mountPath: /etc/config-logging
         - name: config-registry-cert
           mountPath: /etc/config-registry-cert
-        # Mount secret for trusted resources
-        - name: verification-secrets
-          mountPath: /etc/verification-secrets
-          readOnly: true
```

Comment:
This was introduced to mount secrets(contains public keys) and configmap could refer to the secret.